internal/mocks/discovery/discovery.go

```diff
@@ -12,8 +12,7 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
-*/
```

Comment:
Do not run gofumpt on files in `internal/mocks` since they're generated by gomock